ipdb/__main__.py

```diff
@@ -87,6 +87,12 @@ def wrap_sys_excepthook():
 
 
 def set_trace(frame=None, context=3):
+    try:
+        # Try to get and redeclare context with value of env variabe
+        context = os.environ['IPDB_CONTEXT_SIZE']
+    except KeyError:
+        pass
```

Comment:
Hmmm.... I'm slightly worried about this behavior.  So, if I have my env var set, then I do:  `set_trace(context=7)`, the argumet gets ignored?  It seems the argument should have precedence over an env var.

An alternative option may be:

```python
def set_trace(frame=None, context=None):
    if context is None: 
        if os.getenv('IPDB_CONTEXT_SIZE') is not None:
            context = int(os.getenv('IPDB_CONTEXT_SIZE'))
        else:
            context = 3
    
```